doc/README.md

```diff
@@ -236,6 +236,7 @@ the request body:
   - `eof`: end the multipart request
   - `{file, Path}`: to stream a file
   - `{file, Path, ExtraHeaders}`: to stream a file
+  - `{file, Path, Name, ExtraHeaders}` : to stream a file with DOM element name
```

Comment:
this change should be done in the `overview.edoc` file. the README is generated from it.

Reply:
fixed